src/test/java/core/InstallWizardTest.java

```diff
@@ -90,7 +90,9 @@ public void wizardInstallCustomPluginsTest() throws IOException {
         // Customize Jenkins step
         WizardCustomizeJenkins wizardCustomize = new WizardCustomizeJenkins(jenkins);
         wizardCustomize.doSelectPluginsToInstall();
-        wizardCustomize.deselectAll();
+        wizardCustomize.selectSuggested();
+        wizardCustomize.selectAll();
+        wizardCustomize.selectNone();
```

Comment:
?  this is clicking everything suggested (which is already checked IIRC?), then everything, then nothing, but the test only wants the nothing.  Why?

there is no point doing this as there are no assertions.  if you want to test the buttons work as expected please assert side effects.